main/ertp/guide/create-specific-erights.md

```diff
@@ -0,0 +1,87 @@
+# Creating a specific eright with ERTP
+
+## Definition of "specific"
+
+In the context of ERTP and assets, "specific" means "non-fungible".
+
+Assets of the same assay are not fungible if each different item of the assay
+cannot be replaced with another.
+House ownership rights are not fungible, because each ownership right refers to a 
```

Comment:
Specificity is a complicated idea to express. I think "house ownership rights" is an uncommon phrase in English even though it makes sense, so maybe we should use a different example. How about something like:

Assets are fungible if each asset is just as valuable as other assets of the same kind. For instance, dollar bills are fungible because it doesn't matter which bill you use. Each dollar bill is interchangeable with any other. Paintings, on the other hand, are not fungible. Paintings have drastically different values depending on the content and painter. A deal involving a specific painting would not be satisfied by another painting.